features/git-repo/bitbucket.feature

```diff
@@ -1,16 +1,16 @@
 Feature: git-repo when origin is on Bitbucket
 
   Scenario Outline: result
-    Given my remote origin is on Bitbucket through <PROTOCOL>
+    Given my remote origin is <ORIGIN>
     And I have "open" installed
     When I run `git repo`
     Then I see the homepage of my Bitbucket repository in my browser
```

Comment:
```
Then I see the Bitbucket homepage of the "<REPOSITORY>" repo in my browser.
```

I think the git-repo steps should also specify the repository similar to git-pr. Thus same for the github and multi platform support